e2e/framework/utils/k8s_helper.go

```diff
@@ -0,0 +1,253 @@
+package utils
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/jmoiron/jsonq"
+	"html/template"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+)
+
+type K8sHelper struct {
+}
+
+func CreatK8sHelper() *K8sHelper {
+	return &K8sHelper{}
+}
+
+func (k8sh *K8sHelper) getMonIp(mon string) (string, error) {
+	//kubectl -n rook get pod mon0 -o json|jq ".status.podIP"|
+	cmdArgs := []string{"-n", "rook", "get", "pod", mon, "-o", "json"}
+	out, err, status := ExecuteCmd("kubectl", cmdArgs)
+	if status == 0 {
+		data := map[string]interface{}{}
+		dec := json.NewDecoder(strings.NewReader(out))
+		dec.Decode(&data)
+		jq := jsonq.NewQuery(data)
```

Comment:
using json path query used simple to me, because I am used to it. I will make this down for future refactor as well.